docs/tutorials/additional-resources.txt

```diff
@@ -0,0 +1,121 @@
+********************
+Additional Resources
+********************
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Mongoid has an extensive ecosystem. Some of the third-party guides and
+blog posts about Mongoid, as well as libraries integrating with or built
+on top of Mongoid, are listed below.
+
+Screencasts
+===========
+
+- `RailsCasts: Mongoid (revised)
+  <https://www.youtube.com/watch?v=L0RqU2MdqXU>`_
+
+  An overview of Mongoid, by Ryan Bates including the basics 
+  of setting up an app, querying for records, adding embedded 
+  associations, overriding the id, and more.
+
+- `Ruby on Rails Web Services and Integration with MongoDB, Week 3: Mongoid
+  <https://www.youtube.com/watch?v=9LylgiMYsUM>`_
+
+  A detailed introduction to Mongoid and Ruby on Rails web services
+  
+- `Create a search bar in Rails with Mongoid
+  <https://www.youtube.com/watch?v=zusWR8jS5-A>`_
+
+  A Tutorial explaining how to implement text search with Mongoid
+
+Articles
+========
+
+- `Setting up Ruby on Rails 5 App with MongoDB in 10 Minutes
+  <https://www.botreetechnologies.com/blog/setting-up-ruby-on-rails-5-app-with-mongodb-in-10-minutes>`_
+
+- `A Simple Content Management System in Sinatra <https://www.sitepoint.com/a-simple-content-management-system-in-sinatra>`_
+
+  Building a content management application with Sinatra and Mongoid
+
+- `How To Create A Ruby API With Sinatra <https://x-team.com/blog/how-to-create-a-ruby-api-with-sinatra>`_
+
+  Creating a Sinatra API with Mongoid
+
+- `Converting an existing Ruby on Rails application to MongoDB <https://ibraheem.ca/posts/convert-rails-to-mongodb>`_
+
+  How to Convert an existing Ruby on Rails application to use MongoDB and Mongoid
+  
+
+Projects
+========
+
+- `Mongoid Demo <https://github.com/mongoid/mongoid-demo>`_
+
+  A repo containing sample applications using Mongoid.
+
+- `Workarea Commerce <https://github.com/workarea-commerce/workarea>`_
+
+  Workarea is an enterprise-grade Ruby on Rails commerce platform that uses Mongoid
+
+
+Libraries
+=========
+
+- `Carrierwave Mongoid <https://github.com/carrierwaveuploader/carrierwave-mongoid>`_
+
+  Mongoid Support for the Carrierwave file uploads gem
+
+- `Mongoid RSpec <https://github.com/mongoid/mongoid-rspec>`_
+
+  RSpec matchers and macros for Mongoid.
```

Comment:
Suggest "for Mongoid applications" since Mongoid itself doesn't need or use these matchers and macros.